verticapy/tests_new/plotting/base_test_files.py

```diff
@@ -300,12 +307,6 @@ def create_plot(self):
             {"columns": self.COL_NAME_VDF_1},
         )
 
-    def col_name_param():
```

Comment:
There was a pylint error as follows:
`verticapy/tests_new/plotting/base_test_files.py:303:4: E0211: Method 'col_name_param' has no argument (no-method-argument)`
Adding `@staticmethod` decorator helped with pylint, but caused failure at the time of running the unit tests. The easiest fix was moving the function out of the class scope.